core/domain/prod_validation_jobs_one_off.py

```diff
@@ -3008,6 +3010,91 @@ def _get_external_id_relationships(cls, item):
                 'exp_ids', exp_models.ExplorationModel, [item.id])]
 
 
+class ExplorationMathRichTextInfoModelValidator(BaseModelValidator):
+    """Class for validating ExplorationMathRichTextInfoModel."""
+
+    @classmethod
+    def _get_external_id_relationships(cls, item):
+        return [
+            ExternalModelFetcherDetails(
+                'exp_ids', exp_models.ExplorationModel, [item.id])]
+
+    @classmethod
+    def _validate_latex_values(
+            cls, item, field_name_to_external_model_references):
+        """Validate that question_content model is equal to
+        QuestionModel.question_state_data.content.html.
+
+        Args:
+            item: ndb.Model. QuestionSummaryModel to validate.
+            field_name_to_external_model_references:
+                dict(str, (list(ExternalModelReference))).
+                A dict keyed by field name. The field name represents
+                a unique identifier provided by the storage
+                model to which the external model is associated. Each value
+                contains a list of ExternalModelReference objects corresponding
+                to the field_name. For examples, all the external Exploration
+                Models corresponding to a storage model can be associated
+                with the field name 'exp_ids'. This dict is used for
+                validation of External Model properties linked to the
+                storage model.
+        """
+        exploration_model_references = (
+            field_name_to_external_model_references['exp_ids'])
+
+        for exploration_model_reference in exploration_model_references:
+            exploration_model = exploration_model_reference.model_instance
```

Comment:
Add a check for missing/deleted model as well as in other validators.

Reply:
I think the  [line](https://github.com/oppia/oppia/pull/9889/files#diff-f4f5c0de40e1267f463d93fa80583973R6219) does that.
`test_missing_exp_model_failure()`

Reply:
I meant add a check here: `if exploration_model_reference is None or exploration_model_reference.deleted` since in that case the job will error out. Example: https://github.com/oppia/oppia/blob/develop/core/domain/prod_validation_jobs_one_off.py#L1072

Reply:
done